model/tensorflow_hub/setup.py

```diff
@@ -0,0 +1,88 @@
+# import os
```

Comment:
This is of course `WIP` at this point. But I'm leaving this note here so we remember to either go back to the `setup.py` and `setup_common.py` approach. Or just stick with `setup.py` (which is fine, same code after all).